### ISSUE TITLE
FunctionOnlyReturningConstant false positive for interfaces with default implementations

### ISSUE THREAD

**Issue by scottkennedy** [arturbosch/detekt#1027]
## Expected Behavior
This should not trigger the `FunctionOnlyReturningConstant` rule.
```
interface InterfaceFunctionReturningConstant {
 	fun interfaceFunctionWithImplementation() = 1
}
```

## Current Behavior
It tells me I should just declare a constant.  `isOpen()` returns `false` on the `KtNamedFunction`.

## Steps to Reproduce (for bugs)
Failing test case: https://github.com/scottkennedy/detekt/commit/58691e872b1cbb5731bb1071f1b80be5bd35aca6

I couldn't quickly figure out how to identify an interface like this, or I would have just fixed it.

**Comment by schalkms** [comment]
That should be excluded. I'm working on it.
Thanks for providing a test code.

**Comment by vanniktech** [comment]
@schalkms this can be closed, I assume?
